src/pkg/services/m365/sites_test.go

```diff
@@ -216,3 +222,146 @@ func (suite *siteUnitSuite) TestGetAllSites() {
 		})
 	}
 }
+
+func userIdentitySet(userID string, userEmail string) models.IdentitySetable {
+	user := models.NewIdentitySet()
+	userIdentity := models.NewUserIdentity()
+	userIdentity.SetId(ptr.To(userID))
+
+	if len(userEmail) > 0 {
+		userIdentity.SetAdditionalData(map[string]any{
+			"email": userEmail,
+		})
+	}
+
+	user.SetUser(userIdentity)
+
+	return user
+}
+
+func groupIdentitySet(groupID string, groupEmail string) models.IdentitySetable {
+	groupData := map[string]any{}
+	if len(groupEmail) > 0 {
+		groupData["email"] = groupEmail
+	}
+
+	if len(groupID) > 0 {
+		groupData["id"] = groupID
+	}
+
+	group := models.NewIdentitySet()
+	group.SetAdditionalData(map[string]any{"group": groupData})
+
+	return group
+}
+
+func dummySite(owner models.IdentitySetable) models.Siteable {
```

Comment:
This should preferably live in /api/mocks/site.go, or /api/testdata/site.go (though that package doesn't exist yet) so that it's easily accesible throughout the codebase.